README.md

```diff
@@ -257,6 +257,23 @@ private URI create(URI url, T body) {
 }
 ```
 
+#### Create resource and retrieve its current state
+
+```java
+private URI create(URI url, String body) {
+    return unit.execute(POST, url, body).dispatch(series(),
+            on(SUCCESSFUL).dispatch(sameHeader("Location"),
+                    on("Content-Location").capture(),
+                    any(String.class).capture(location().andThen(location ->
```

Comment:
Yes, wording is an issue. It should read as *when the Location header has the same value as Content-Location, then capture otherwise make another GET request*